test/performance/observed-concurrency/observed_concurrency.go

```diff
@@ -37,6 +41,14 @@ func main() {
 	log.Print("Requests against '/?timeout={TIME_IN_MILLISECONDS}' will sleep for the given time.")
 	log.Print("Each request will return its serverside start and end-time in nanoseconds.")
 
-	http.HandleFunc("/", handler)
-	http.ListenAndServe(":8080", nil)
+	m := http.NewServeMux()
```

Comment:
Can we make this pattern a helper function, so new tests will use it instead of repeating it -- or worse -- using a mechanism that is known to cause flakiness?